scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/CertificateRequest.java

```diff
@@ -388,18 +374,16 @@ private boolean addCerticiateAuthorities(List<X500Principal> authorities) {
 	 *         maximum encoded length allowed for the certificate request message's
 	 *         certificate authorities vector (2^16 - 1 bytes).
 	 */
-	public boolean addCertificateAuthorities(X509Certificate[] trustedCas) {
```

Comment:
[API break] should we also keep the old version ?

Reply:
FMPOV, that's internal API and not considered to be not adapted.